src/ServiceControl.Persistence/EndpointsView.cs

```diff
@@ -11,5 +12,6 @@ public class EndpointsView
         public bool MonitorHeartbeat { get; set; }
         public HeartbeatInformation HeartbeatInformation { get; set; }
         public bool IsSendingHeartbeats { get; set; }
+        [JsonIgnore] public bool IsNotSendingHeartbeats { get; set; }
```

Comment:
can this be readonly negation of IsSendingHeartbeats?

Reply:
I wish, the reason I needed this is because there is an "unknown" state and I want to count